src/pages/docs/deployments/kubernetes/deploy-raw-yaml/index.md

```diff
@@ -82,31 +84,45 @@ To configure an inline YAML source, select the `Inline YAML` as your YAML Source
 This is **not** the recommended approach for advanced cases as it does not allow for version management unless you are using it in conjunction with [Config As Code](/docs/projects/version-control).
 :::
 
+## Referencing packages
+
+Container images can be selected as **[Referenced Packages](/docs/deployments/custom-scripts/run-a-script-step#referencing-packages)** to automatically generate variables referring to the image name and tag that can be substituted in your manifests.
```

Comment:
Might be worth adding something here to note that the image doesn't have to be downloaded for this to work.

Reply:
I'm gonna leave this as is, I don't think anything implies this will or won't download the image and the information likely belongs elsewhere.

I'm pretty sure the UI says it won't be downloaded somewhere too, so I'm not concerned.